Resources/Prototypes/_SpaceCats/Entities/Stations/base.yml

```
@@ -0,0 +1,78 @@
+    - displayName: stock-trading-company-hound
+      basePrice: 187
+      currentPrice: 176
+    - displayName: stock-trading-company-harvest
+      basePrice: 62
+      currentPrice: 55
+    - displayName: stock-trading-company-gasdef
+      basePrice: 250
+      currentPrice: 243
+    - displayName: stock-trading-company-hrc
+      basePrice: 280
+      currentPrice: 211
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)